src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Remote/CodeActionRequestInfo.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Text.Json.Serialization;
+using Microsoft.CodeAnalysis.Razor.Protocol;
+using Microsoft.CodeAnalysis.Razor.Protocol.CodeActions;
+
+namespace Microsoft.CodeAnalysis.Razor.Remote;
+
+internal record CodeActionRequestInfo(
+    [property: JsonPropertyName("languageKind")]
+    RazorLanguageKind LanguageKind,
+    [property: JsonPropertyName("csharpRequest")]
+    VSCodeActionParams? CSharpRequest);
```

Comment:
Did you mean for these parameters to be split across multiple lines? I don't think we've been formatting `JsonPropertyName` attributes like this.

Reply:
Yes it was intentional, as I find this much more readable, and matches the style used when things are actually properties like in the LSP types. Having said that, it does seem that every other instance is the single line style, so I will update this and accept defeat.